test/verify/check-metrics

```diff
@@ -839,65 +839,89 @@ podman save quay.io/libpod/busybox | sudo -i -u admin podman load
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
         # No matching type
-        m.execute("mkdir -p /tmp/hwmon/hwmon0")
-        m.execute("echo BAT0 > /tmp/hwmon/hwmon0/name")
-        m.execute("mount -o bind /tmp /sys/class")
+        m.execute("mkdir -p /tmp/sensor-sys-class/hwmon/hwmon0")
+        self.addCleanup(m.execute, "rm -rf /tmp/sensor-sys-class/hwmon")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon0/name", "BAT0")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon0/temp1_input", "40000")
+        m.execute("mount -o bind /tmp/sensor-sys-class /sys/class")
+        self.addCleanup(m.execute, "umount /sys/class")
 
         b.wait_not_present("#current-metrics-card-cpu .temperature")
 
-        # create files that contain CPU temperature
-        m.execute("mkdir -p /tmp/hwmon/hwmon1")
-
-        # Testing on ARM devices
-        m.execute("echo cpu_thermal > /tmp/hwmon/hwmon1/name")
-        m.execute("echo 60000 > /tmp/hwmon/hwmon1/temp1_input")
-
-        b.wait_in_text("#current-metrics-card-cpu", "60 °C")
+        def clean_and_relog():
+            m.execute("rm -rf /sys/class/hwmon/hwmon1/*")
+            b.relogin()
+            b.go("/metrics")
+            b.enter_page("/metrics")
 
-        # Testing on AMD devices
-        m.execute("echo k10temp > /tmp/hwmon/hwmon1/name")
-        m.execute("echo Tctl > /tmp/hwmon/hwmon1/temp1_label && echo Tccd1 > /tmp/hwmon/hwmon1/temp2_label && echo Tccd3 > /tmp/hwmon/hwmon1/temp3_label")
-        # setting temp1_input to the highest value as Tctl should be ignored
-        m.execute("echo 150000 > /tmp/hwmon/hwmon1/temp1_input && echo 45000 > /tmp/hwmon/hwmon1/temp2_input && echo 43000 > /tmp/hwmon/hwmon1/temp3_input")
+        # create files that contain CPU temperature
+        # ARM
+        m.execute("mkdir -p /tmp/sensor-sys-class/hwmon/hwmon1")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/name", "cpu_thermal")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp1_input", "30000")
 
+        b.wait_in_text("#current-metrics-card-cpu", "30 °C")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp1_input", "45000")
         b.wait_in_text("#current-metrics-card-cpu", "45 °C")
 
-        m.execute("echo 90000 > /tmp/hwmon/hwmon1/temp3_input")
+        clean_and_relog()
+
+        # AMD
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/name", "k10temp")
+        # Tctl (temp1_input) will be ignored
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp1_label", "Tctl")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp2_label", "Tccd1")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp3_label", "Tccd3")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp1_input", "40000")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp2_input", "35000")
+        m.write("/tmp/sensor-sys-class/hwmon/hwmon1/temp3_input", "30000")
```

Comment:
This would be a bit easier to read if you keep the _label and _input of a particular sensor together; i.e. the "40000" will be ignored.